ddocs/medic-client/views/docs_by_id_lineage/map.js

```diff
@@ -0,0 +1,20 @@
+function(doc) {
```

Comment:
`docs_by_id_and_lineage` then? 
`docs_by_id_lineage` makes it look like it's indexed by id-lineage (lineage of ids), which doesn't seem to make sense to me.